packages/node/src/client.ts

```diff
@@ -163,6 +330,34 @@ export abstract class PostHogBackendClient extends PostHogCoreStateless implemen
     )
   }
 
+  /**
+   * Capture an event immediately (synchronously).
+   *
+   * @example
+   * ```ts
+   * // Basic immediate capture
+   * await client.captureImmediate({
+   *   distinctId: 'user_123',
+   *   event: 'button_clicked',
+   *   properties: { button_color: 'red' }
+   * })
+   * ```
+   *
+   * @example
+   * ```ts
+   * // With feature flags
+   * await client.captureImmediate({
+   *   distinctId: 'user_123',
+   *   event: 'user_action',
+   *   sendFeatureFlags: true
```

Comment:
NB – perhaps worth mentioning that `sendFeatureFlags` is of type `boolean | SendFeatureFlagsOptions`, and `SendFeatureFlagsOptions` lets users specify more information about how to send feature flags along with the captured data

```
export interface SendFeatureFlagsOptions {
  onlyEvaluateLocally?: boolean
  personProperties?: Record<string, any>
  groupProperties?: Record<string, Record<string, any>>
  flagKeys?: string[]
}
```

Maybe worth including another example with feature flags using the `SendFeatureFlagsOptions`.